### PR TITLE
fix: update the overview page

### PR DESCRIPTION
**Pull request by Ngozi-Txfusion**
## What

- Update the overview page for web3js
- Structure the pages to follow the contribution guideline

## Why

- Improve Usability and Clarity
- Maintain Consistency
- Enhance Community Contributions

### CODE REVIEW

**Review by petarTxFusion — DISMISSED** [review]

**Review by petarTxFusion — DISMISSED** [review]

**Review by danijelTxFusion — DISMISSED** [review]

File: content/10.js/01.web3js/00.index.md [hunk 1]
```diff
@@ -9,19 +9,82 @@ links:
     target: _blank
 ---
 
+## Web3.js plugin for ZKsync
```
Comment (itsacoyote) [inline content/10.js/01.web3js/00.index.md:12]: Let's remove this header.

File: content/10.js/01.web3js/00.index.md [hunk 2]
```diff
@@ -9,19 +9,82 @@ links:
...
+
 The [Web3.js plugin for ZKsync](https://github.com/ChainSafe/web3-plugin-zksync) makes it easy for developers to
 interact with the ZKsync Era network. See the comprehensive [API documentation](https://chainsafe.github.io/web3-plugin-zksync/)
-for a full description of the plugin's capabilities. The following sections provide explanations and code samples for
-plugin users:
+for a full description of the plugin's capabilities.
+
+## Documentation sections
+
+Below, you’ll find sections that offer explanations for using the plugin:
```
Comment (itsacoyote) [inline content/10.js/01.web3js/00.index.md:20]: Remove this header and line as well.

File: content/10.js/01.web3js/00.index.md [hunk 3]
```diff
@@ -9,19 +9,82 @@ links:
...
+---
+A guide to the various RPC methods available in the plugin.
+::
+::card
+---
+title: Constants, Types, & Utilities
+icon: i-heroicons-variable
+to: /js/web3js/constants-types-utilities
+---
+Information on constants, types, and utility functions provided by the plugin.e.
```
Comment (itsacoyote) [inline content/10.js/01.web3js/00.index.md:45]:
```suggestion
Information on constants, types, and utility functions provided by the plugin.
```

**Review by itsacoyote — CHANGES_REQUESTED** [review]

**Review by petarTxFusion — APPROVED** [review]

**Review by itsacoyote — APPROVED** [review]
Thank you and great catch on the headers capitalizations 👍🏻

**Pull request merged**